### ISSUE TITLE
Arecord not working

### ISSUE THREAD

**Issue by mintu19** [voice-engine/ec#6]
When i try to record mic using arecord with ec, i get error

```
ALSA lib pcm.c:7448:(snd_pcm_slave_conf) Unknown field type
arecord: main:788: audio open error: Invalid argument
```

How to solve this

**Comment by karostami** [comment]
I have a similar mistake. 
Did you correct this error?

**Comment by xiongyihui** [comment]
Can you provide more details? Did you install [the FIFO plugin](https://github.com/voice-engine/ec#use-ec-with-alsa-plugins-as-alsa-devices)?

**Comment by mintu19** [comment]
Yes i had installed Fifo plugin.

**Comment by karostami** [comment]
Yes. I install too, i use in Raspberry zero pi. when I use the pulse audio is everything O.K. But without it, i get error.

**Comment by xiongyihui** [comment]
It seems that the FIFO plugin is broken. Trying to figure it out~

**Comment by xiongyihui** [comment]
The `asound.conf` had an issue. just fixed it.

**Comment by karostami** [comment]
Thanks. It is O.K!
But when I use arecord then stop it, ec is suddenly stoped!

**Comment by mintu19** [comment]
Same issue... Ec stops with ctrl+c on arecord...

Also recorded voice is flat like a playing at 1/2 speed... I will file another issue for that after testing more and if issue persists...
